collectors/cgroups.plugin/sys_fs_cgroup.c

```diff
@@ -6,6 +6,12 @@
 #define PLUGIN_CGROUPS_MODULE_SYSTEMD_NAME "systemd"
 #define PLUGIN_CGROUPS_MODULE_CGROUPS_NAME "/sys/fs/cgroup"
 
+#ifdef NETDATA_INTERNAL_CHECKS
+#define CGROUP_PROCFILE_FLAG PROCFILE_FLAG_NO_ERROR_ON_FILE_IO
+#else
+#define CGROUP_PROCFILE_FLAG PROCFILE_FLAG_DEFAULT
+#endif
```

Comment:
should be opposite - no errors by default

Reply:
Thanks! Done.